src/plugins/imagery/components/ImageryViewLayout.vue

```diff
@@ -262,6 +331,69 @@ export default {
 
             this.time = this.timeFormat.format(datum);
             this.imageUrl = this.imageFormat.format(datum);
+        },
+        getFormatter(key) {
+            let valueFormatter = this.openmct.telemetry.getValueFormatter({
+                format: key
+            });
+
+            return valueFormatter.formatter;
+        },
+        numericImageAge() {
+            let currentTime = this.currentTimeValue();
+            let selectedImage = this.getSelectedImage();
+            let formatter = this.getFormatter(this.timeKey);
```

Comment:
We need to cache the result of this, it's being evaluated every 100ms (x 8 cameras for VIPER). It will only change if the time system changes,so it only needs to be evaluated then.

Reply:
Hey I set selected image and formatter to to computed values, then also set numericImageAge to a computed value, does this solve the issue you were seeing?